### PR TITLE
Implement API/UI for querying chain head/wasm 

### PR DESCRIPTION
**Pull request by BulatSaif**
Fixes: https://github.com/paritytech/testnet-manager/issues/44
Relay chain runtime: 
image

Parachain page: 
Offboarded parachain:
image
Onboarded parachain: 
image

API:
image

### CODE REVIEW

File: app/lib/network_utils.py [hunk 1]
```diff
@@ -682,3 +683,33 @@ async def set_collator_nodes_keys_on_chain(para_id, nodes=[], statefulset=''):
     for node_name in nodes:
         ss58_format = get_substrate_node(node_name).get("ss58_format")
         collator_set_keys(node_name, para_id, ss58_format)
+
+
+def get_substrate_runtime(node_client):
+    last_runtime_upgrade = node_client.query("System", "LastRuntimeUpgrade").value
+    wasm = get_parachain_wasm(node_client)
```
Comment (PierreBesson) [inline app/lib/network_utils.py:690]: I think the function `get_parachain_wasm` can be renamed `get_chain_wasm` as it is actually agnostic of what kind of chain it is.

**Comment by PierreBesson** [comment]
I think we also need add a link to these pages:
- link to relay-chain runtime on the main page
- link to each parachain runtime page on the parachain overview replacing those last 2 columns

img drawio

**Comment by BulatSaif** [comment]
> I think we also need add a link to these pages:
> 
> * link to relay-chain runtime on the main page
> * link to each parachain runtime page on the parachain overview replacing those last 2 columns
> 
> img drawio

I think we should  replace `Head` and `CurrentCodeHash` columns with `Runtime`  column, which will have links to runtime page

**Comment by PierreBesson** [comment]
> I think we should replace Head and CurrentCodeHash columns with Runtime column, which will have links to runtime page

Yes that was exactly my idea.

**Review by PierreBesson — APPROVED** [review]

**Comment by BulatSaif** [comment]
Added links to parachain page:
image

**Pull request merged**